server/routes/article.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const router = express.Router();
+const articleLibs = require('../libs/articlelibs');
+const {authenticate} = require('../middleware/authenticate');
+
+router.post('/article/', (req, res) => {
+    let data = {
+        topic : req.body.topic,
+        article : req.body.article,
+        uuid : req.header('uuid')
+    }
+    articleLibs.createArticle(data).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/article/:articleId', (req, res) => {
+    let query = {
+        articleId : req.params.articleId
+    }
+    articleLibs.getArticle(query).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.get('/article/:topic', (req, res) => {
+    let query = {
+        topic : req.params.topic
+    }
+    articleLibs.getArticle(query).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+});
+
+router.patch('/article/addComment', (req, res) => {
+    let data = {
+        comment : req.body.comment,
+        articleId : req.body.articleId,
+        uuid : req.header('uuid')
+    }
+    articleLibs.addComment(data).then((article) => {
+        res.status(article.status).send(article);
+    }).catch((error) => {
+        res.status(error.status).send(error);
+    });
+})
+
+router.patch('/article/editComment', (req, res) => {
+    let data = req.body;
```

Comment:
no need for this variable just pass directly req.body in the function

Reply:
Done